stores/sql_test.go

```diff
@@ -40,11 +42,12 @@ type testSQLStoreConfig struct {
 	dbName          string
 	dbMetricsName   string
 	dir             string
+	ephemeral       bool
```

Comment:
I'd invert this to `persistent` to keep the nice behaviour of every field defaulting to the defaults in `defaultTestSQLStoreConfig` if not set.